config.json

```diff
@@ -1564,6 +1564,16 @@
       "topics": [
         "arrays"
       ]
+    },
+    {
+      "slug": "knapsack",
+      "uuid": "b0301d0b-d97a-4043-bd82-ba1edf8c1b16",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
```

Comment:
This is definitely more difficult than 1. At a glance, I would recommend a 5 or 6, as `change` is a 4, and this is fundamentally a slightly-harder version of that exercise.